### DISCUSSION TITLE
[DONE] Allow auto-merge in edit mode using existing Blender auto-merge toggle

### DISCUSSION THREAD

**Discussion by JulienHeijmans** [JulienHeijmans/quicksnap#19] [Potential Upcoming features]
I am considering to use the auto-merge toggle from Blender's edit mode:
![image](https://user-images.githubusercontent.com/35562774/196239196-c93012b4-58dd-47bc-9400-87fb4f81377e.png)

When that toggle is enabled and the user snap points in edit mode, the points would automatically merge with other points if they are snapped on top of each other.

I am a little worried that some users might not know about that toggle and merge points unintentionally. Any thought?

**Comment by Aritaborian** [comment]
Let user intentionally choose this options in addon interface (synchronizing it with native button in Blender interface) for make it clearly visible in any situation.
